infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -423,51 +423,52 @@ EOT
-click on `Third Party Access`. Click on `Configure` the application you have installed in previous step and you will find the `installationId` at the URL of the browser:
+     Go to the "Developer Settings" section of your organization. Then, click on "Edit" for your "Github App" and once you are in the app settings, click on "Install App" and click on the "Install" button. Accept the permissions to install it to your whole organization.
+  5. Once installed, the `installationId` is required as it needs to be provided in the Psoxy as parameter for the connector in your Terraform module. You can go to your organization settings and
+click on `Third Party Access`. Click on `Configure` the application you have installed in the previous step and you will find the `installationId` at the URL of the browser:
 ```
 https://github.com/organizations/{YOUR ORG}/settings/installations/{INSTALLATION_ID}
 ```
-  Copy the value of `installationId` and assign it to the `github_installation_id` variable in Terraform. You will need to redeploy the proxy again if that value was not populated before.
+  Copy the value of `installationId` and assign it to the `github_installation_id` variable in Terraform. If the variable wasn't defined before, add the following to your `terraform.tfvars` file
+  `github_installation_id="{installationId}"` replacing "installationId" by the the actual value. You will need to redeploy the Psoxy again for the changes to take effect.
 
 **NOTE**:
  - If `github_installation_id` is not set, authentication URL will not be properly formatted and you will see *401: Unauthorized* when trying to get an access token.
- - If you see *404: Not found* in logs please review the *IP restriction policies* that your organization might have; that could cause connections from psoxy AWS Lambda/GCP Cloud Functions be rejected.
+ - If you see *404: Not found* in logs please review the *IP restriction policies* that your organization might have; that could cause connections from the Psoxy AWS Lambda/GCP Cloud Functions be rejected.
 
   6. Update the variables with values obtained in previous step:
-     - `PSOXY_GITHUB_CLIENT_ID` with `App ID` value. **NOTE**: It should be `App Id` value as we are going to use authentication through the App and **not** *client_id*.
-     - `PSOXY_GITHUB_PRIVATE_KEY` with content of the `gh_pk_pkcs8.pem` from previous step. You could open the certificate with VS Code or any other editor and copy all the content *as-is* into this variable.
+     - `${var.config_parameter_prefix}GITHUB_CLIENT_ID` with `App ID` value. **NOTE**: It should be `App Id` value as we are going to use authentication through the App and **not** *client_id*.
+     - `${var.config_parameter_prefix}GITHUB_PRIVATE_KEY` with content of the `gh_pk_pkcs8.pem` from previous step. You could open the certificate with VS Code or any other editor and copy all the content *as-is* into this variable.
```

Comment:
Yes, this is something we need to solve but it's complicated. The value you want here is specific to each proxy deployment.  

By default, it's `PSOXY_GITHUB_` for the whole thing; but this is just what's for the default case. 

Both the `PSOXY_` prefix can differ and the `GITHUB_` prefix for the lambda (cloud function).
  - The former is to support customers putting the SSM parameters in some hierachy, esp when deploying to a shared AWS account; they might use `people-analytics/worklytics/` for example in AWS case (which conventionally uses /-style hierarchies).  
  - The latter may vary if multiple instances of GitHub connectors for example, which we want/need to support.  in which case they may have several distinct proxy deployments of the same source kind. (eg, we have people who have multiple GitHub organizations, such as one for their open source stuff, and another one for in-house)

I've tried to solve, but the logic that we use to generate the prefix value is in the `aws-psoxy-rest`/`gcp-psoxy-rest` modules atm, and as I recall I get cyclic dependency if I try to output from there to fill as a variable to `worklytics-connectors-` modules.  Avoiding that may be a bigger refactor - splitting out the logic that defines the identifiers for each proxy instance independently at the top level of our examples, at the cost of adding an additional clone of the repository to the dependency footprint. 

Other approach is to leave this parameterized, and the value of `external_token_todo` is a template to be evaluated later, once the